.github/workflows/ci.yml

```diff
@@ -202,6 +202,25 @@ jobs:
         run: yarn test-unit
       - name: Integration Tests
         run: yarn test-integration
+  test-snowflake:
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [10, 12]
```

Comment:
Please change the highest tested node version from 12 to 16, we recently changed this